pay/lib/src/widgets/pay_button.dart

```diff
@@ -51,16 +51,31 @@ abstract class PayButton extends StatefulWidget {
   /// Initializes the button and the payment client that handles the requests.
   PayButton(
     Key? key,
-    String paymentConfigurationAsset,
+    String? paymentConfigurationAsset,
+    String? paymentConfigurationString,
     this.onPaymentResult,
     this.width,
     this.height,
     this.margin,
     this.onError,
     this.childOnError,
     this.loadingIndicator,
-  )   : _payClient = Pay.withAssets([paymentConfigurationAsset]),
-        super(key: key);
+  ) : super(key: key) {
+    if (paymentConfigurationAsset != null &&
```

Comment:
I feel that the package would do fine without protecting for this scenario. Most examples use assets, and for those needing to use strings, it seems explicit enough that only one of the two need to be provided.
I suggest we leave this check out and reconsider it again if we see users having issues after setting both parameters.